static/src/stylesheets/module/identity/_wizard.scss

```diff
@@ -22,3 +51,25 @@
         display: none;
     }
 }
+
+@keyframes manage-account-wizard__fadeLeft {
+    from {
+        transform: translateX(0);
+        opacity: 1;
+    }
+    to {
+        transform: translateX(-80px);
```

Comment:
can you leave a comment to explain where `80` comes from?

Reply:
thing is, it comes from nowhere, it's a arbitrary number that more or less doesn't look too fast for a .2s transition? i can explain that i guess